src/spring/azext_spring/_client_factory.py

```diff
@@ -32,6 +32,14 @@ def cf_bindings(cli_ctx, *_):
     return cf_spring(cli_ctx).bindings
 
 
+def cf_eureka_servers(cli_ctx, *_):
+    return cf_spring(cli_ctx).eureka_servers
+
+
+def cf_eureka_servers(cli_ctx, *_):
+    return cf_spring(cli_ctx).eureka_servers
```

Comment:
Whether is it the same as above? If so, please remove it.

Reply:
Removed